morpheus/pipeline/receiver.py

```diff
@@ -71,58 +72,122 @@ def is_partial(self):
 
     @property
     def in_pair(self):
-        return (self.in_stream, self.in_pair)
+        return (self.in_node, self.in_pair)
```

Comment:
Important: Delete this property. It's not being used. If it were, it would cause infinite recursion.

Reply:
good catch, looks like it was never used.